### DISCUSSION TITLE
我提了一个PR，给了这么一个错误“1 lines in your changes are missing coverage. Please review”

### DISCUSSION THREAD

**Discussion by gb853940223** [ant-design/ant-design#47088] [Q&A]
提了一个PR，但是github报出了这么一个问题。看上去像是单测的问题，但是不太懂要怎么处理。
![image](https://github.com/ant-design/ant-design/assets/24557533/4842c632-b54e-45d9-9302-42126f9d41e3)
![image](https://github.com/ant-design/ant-design/assets/24557533/438b7b15-ed41-4046-8d9c-e023d7bf8341)
遇到这种问题，请问大佬们要怎么处理。

**Comment by afc163** [comment]
需要增加测试用例，让用例能跑到这一行。
